server/src/routes/api/v1/shoesRouter.js

```diff
@@ -8,6 +8,18 @@ const shoesRouter = new express.Router()
 
 shoesRouter.use("/:shoeId/reviews", shoeReviewsRouter)
 
+shoesRouter.delete("/:id", async (req, res) => {
+  const { body } = req
+  try {
+    const shoe = await Shoe.query().findById(body.id)
+    const deletedReviews = await shoe.$relatedQuery("reviews").delete()
+    const deletedShoe = await Shoe.query().findById(body.id).delete()
+    return res.status(202).json({ deleted: deletedShoe })
```

Comment:
204 would be a more appropriate action - 202 indicates "Accepted", but the action (delete) has not yet completed. It's like saying "we queued up the shoe to be deleted but it hasn't finished yet"